frontend/src/components/Schedules/Schedules.util.ts

```diff
@@ -19,6 +25,147 @@ export const formatTime = (time: string) => {
   return `${hourPrefix} ${hour % 12 || 12}시`;
 };
 
+export const generateTimeRange = (firstTime: string, lastTime: string): number[] => {
+  const start = parseInt(firstTime.slice(0, 2));
+  const end = parseInt(lastTime.slice(0, 2));
+
+  return Array.from({ length: end - start }, (_, index) => {
+    const hour = start + index;
+    return hour;
+  });
+};
```

Comment:
[P3]

요렇게 줄여볼 수도 있을 것 같네요! 명시적으로 나타내기 위해 변수를 사용한것이겠죠..?

```suggestion
  return Array.from({ length: end - start }, (_, index) => start + index);
};
```

Reply:
네넵 가독성을 위해서 만든 변수입니다! 

짧고 간단한 코드보다는 읽는 사람이 편한 즉, 가독성이 좋은 코드가 빛을 낼 때도 있으니까요!! ✨✨✨